app/controllers/v1/kitchen_joins_controller.rb

```diff
@@ -1,49 +1,48 @@
 # frozen_string_literal: true
 
+require 'securerandom'
+
 module V1
   class KitchenJoinsController < ApplicationController
-    api :POST, '/v1/kitchen_joins', 'Create kitchen joins'
-    param :user_code, String, required: true, desc: 'code of a user to invite'
-    def create
-      user = User.find_by!(code: params[:user_code])
-
-      kitchen_join = KitchenJoin.new(
-        user: user,
-        kitchen: @current_user.kitchen,
-        expired_at: Time.zone.now + 3.minutes
-      )
-
-      kitchen_join.save!
+    # 招待者用メソッド
+    api :GET, '/v1/kitchen_joins/show_code', 'Get the current user\'s invitation code or create one if it does not exist'
```

Comment:
多分 " ... " じゃないと、エスケープできなそう
```suggestion
    api :GET, '/v1/kitchen_joins/show_code', "Get the current user\'s invitation code or create one if it does not exist"
```